contracts/FractalUsul.sol

```diff
@@ -0,0 +1,56 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Usul was previously named "Seele" and SekerDAO was TokenWalk
+// that's where this naming differences are coming from
+import "@tokenwalk/seele/contracts/Usul.sol";
+
+contract FractalUsul is Usul {
+  struct Transaction {
+    address to;
+    uint256 value;
+    bytes data;
+    Enum.Operation operation;
+  }
+
+  event ProposalMetadataCreated(
+    uint256 proposalId, 
+    Transaction[] transactions,
+    string title,
+    string description, 
+    string documentationUrl
+  );
+
+  /// @dev This method is used instead of Usul.submitProposal. Essentially - it just implements same behavior
+  /// but then - it also emits metadata of the proposal in ProposalMetadataCreated event.
+  function submitProposalWithMetaData(
+        Transaction[] memory transactions,
+        bytes32[] memory txHashes,
+        address strategy,
+        bytes memory data,
+        string calldata title,
+        string calldata description,
+        string calldata documentationUrl
+  ) external {
```

Comment:
@decent-dao/fractal for reference, this is mostly identical to the Usul `submitProposal` function:

https://github.com/SekerDAO/Usul/blob/0cb39c0dd941b2825d401de69d16d41138a26717/contracts/Usul.sol#L216

The reason it's copied here is because that function can't be extended, and is also `external`, so can't just be called internally here

Reply:
@mudrila I assume not since we can't override it, but is there any way we can block/revert calls to the parent `createProposal`?

Reply:
@herbig I was thinking about this today. Unfortunately we can't prevent someone from submitting a proposal through that function, since it can't be overriden. However, we should be able to modify the UsulVetoGuard so that it doesn't allow proposals to be executed that weren't created with this new function.

Reply:
yeah, I think that's fine, it's not worth preventing @tbwebb22 

If users approve a proposal that they don't know anything about, that's on them ha